token-bucket.cabal

```diff
@@ -27,19 +27,19 @@ flag use-cbits
 
 library
   default-language:    Haskell2010
-  build-depends:       base >=4.6 && <4.9
+  build-depends:       base >=4.6 && <4.10
   hs-source-dirs:      src
   if flag(use-cbits)
     cpp-options:       -DUSE_CBITS
     c-sources:         cbits/get_posix_time.c
   else
-    build-depends:     time >=1.4 && <1.6
+    build-depends:     time >=1.4 && <1.8
```

Comment:
The upper bound on `time` seems wrong; there is no `time-1.7` or later release afaik